ita-social-projects/StreetCode_Client
# Allow multiple file uploads in art gallery

How to check if it's working: 
1. Log in as admin (/admin-panel url)
2. Click on edit button on any existing street code
3. Scroll down to Art gallery 
4. Click + button -> modal with file provider will open
5. Select multiple files and click OK
6. ALL this files will appear in art gallery 
7. When you click "зберегти як чернетку" page will reload and you will see your uploaded images in art gallery block.

* If you can not select multiple file in modal notify me please
